pkgs/development/haskell-modules/configuration-ghc-9.2.x.nix

```diff
@@ -102,11 +102,6 @@ self: super: {
   # https://github.com/protolude/protolude/issues/127#issuecomment-1428807874
   protolude = doJailbreak super.protolude;
 
-  # https://github.com/fpco/inline-c/pull/131
-  inline-c-cpp =
-    (if isDarwin then appendConfigureFlags ["--ghc-option=-fcompact-unwind"] else x: x)
-    super.inline-c-cpp;
```

Comment:
9.2 still needs it afaik.

Reply:
It's applied unconditionally in `configuration-darwin.nix` and I think all versions need it? (Well, GHC >= 9.4 is broken for unrelated reasons.)

Reply:
I overlooked that, but it's the default behavior for 9.4, so I'd prefer not to set it in 9.4 and above.
I guess this comment applies to that line now.

Reply:
Okay that means we just have to drop the override for GHC HEAD and leave the rest as is.